Doc/library/math.rst

```diff
@@ -136,6 +136,15 @@ Number-theoretic and representation functions
    .. versionadded:: 3.5
 
 
+.. function:: lcm(a, b)
+
+   Return the least common multiple of integers *a* and *b*.  If both *a* or
```

Comment:
"If both *a* or *b* is nonzero" isn't grammatical. How about:

"The value of ``lcm(a, b)`` is the smallest nonnegative integer that is a multiple of both *a* and *b*."

There's no particularly good reason to specify that ``lcm(0, 0)`` is ``0``, since ``lcm(0, 0)`` is really no more special than ``lcm(0, 1)`` or ``lcm(1, 0)``, which are also both ``0``.

To reassure those worried about what "multiple of" means in the presence of zeros, I'd suggest a further sentence:

"If either *a* or *b* is zero then ``lcm(a, b)`` is zero."